java/kotlin-jmod/src/main/java/com/google/protobuf/kotlin/ProtobufKotlinStub.java

```diff
@@ -0,0 +1,4 @@
+package com.google.protobuf.kotlin;
+
+/** Invisible class for JPMS build. */
+final class ProtobufKotlin {}
```

Comment:
This stub class must be present to make the package `com.google.protobuf.kotlin` show up during the `javac`-only compile; otherwise, the `javac` execution would need to be aware of Kotlin class outputs. It is empty, final, and should be compiled away by `javac` anyway since it is not used anywhere. Should not be visible to users.